spec/rails_helper.rb

```diff
@@ -9,9 +9,25 @@
 
 require 'capybara-screenshot/rspec'
 
-# Poltergeist with PhantomJS
-require 'capybara/poltergeist'
-Capybara.javascript_driver = :poltergeist
+# Headless Chrome
+require 'selenium/webdriver'
+
+Capybara.register_driver :chrome do |app|
+  Capybara::Selenium::Driver.new(app, browser: :chrome)
+end
+
+Capybara.register_driver :selenium do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: { args: %w(headless disable-gpu window-size=1366,768) }
```

Comment:
What's the significance of the window size?

Reply:
Chrome was defaulting to a smaller window size that caused Bootstrap to display the topbar  in "hamburger mode" so tests that looked for "Staff" to appear in the main menu would fail because "Staff" wasn't visible.